swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+# Wave 5
+#=========================================  
+class Clothing(Item):
+    def __init__(self, condition=0):
+        super().__init__("Clothing", condition)
```

Comment:
To be consistent with the constructor in other items (Decor and Electronics), we can set our arguments explicitly: 
```
super().__init__(category = "Clothing", condition = condition)
```
Or we could edit Decor and Electronics to followed the ordered parameters pattern!